packages/tinacms/src/toolkit/forms/form.ts

```diff
@@ -495,13 +495,13 @@ export class Form<S = any, F extends Field = AnyField> implements Plugin {
                 {
                   type: 'string',
                   label: 'Alt',
-                  name: [templateName, 'alt'].join('.'),
+                  name: [templateName.replace(/\.props$/, ''), 'alt'].join('.'),
                   component: 'text',
                 },
                 {
                   type: 'string',
                   label: 'Caption',
-                  name: [templateName, 'caption'].join('.'),
+                  name: [templateName.replace(/\.props$/, ''), 'caption'].join('.'),
```

Comment:
While this works, it is an ugly fix. I expect the maintainer will want to try rework this code a bit.

Reply:
lgtm